### DISCUSSION TITLE
`Spin` 的全屏模式为什么不用 `Portal` 丢到 `body` 出去呢？

### DISCUSSION THREAD

**Discussion by xsjcTony** [ant-design/ant-design#46940] [Ideas]
RT，我的父元素中有用到 `backdrop-filter`, 直接导致样式拉闸。这种全屏的如果要用 `fixed`, 应该要portal到外面才对吧？
![image](https://github.com/ant-design/ant-design/assets/45784210/b3b23440-a5f9-4bb2-807f-4137d9aaafce)

我用 `createPortal()` 把他丢出去才解决的 感觉默认这样会比较好？或者至少像tooltip一样提供一个 `getPopupContainer` 会比较好?

**Comment by li-jia-nan** [comment]
这个需要先把 `rc-dialog` 和 `rc-drawer` 合并，再提供 Spin 的能力，后面大版本会支持，目前只是用 css 先写了一个动画简单实现了

**Reply by xsjcTony** [reply]
无所谓不着急 反正我能自己丢出去

**Reply by li-jia-nan** [reply]
自己需要支持的话，可以先封装一个 HOC 实现一下

**Comment by afc163** [comment]
@li-jia-nan 这个和 dialog 有啥关系？感觉可以直接在 spin 内实现就行了。

**Reply by li-jia-nan** [reply]
我记得是 `rc-dialog` 内部有 `ReactDom.createPortal` 的逻辑，我们 `Modal` 组件底层也用了这个能力，如果 Spin 要支持 createPortal 能力的话，肯定是复用 `rc-dialog` 的能力了，不会单独实现，之前有和闲夕大佬讨论过这个，闲夕大佬当时这么回复的

**Reply by afc163** [reply]
createPortal 可以单独用吧，感觉也不复杂。

**Reply by li-jia-nan** [reply]
可以单独用，但是这个我拿不准了，需要 @MadCcc @zombieJ 大佬们决定一下
